src/g_cal.py

```
@@ -50,14 +50,15 @@ def get_tasks_service(creds: Credentials):
 
 
 
-def list_all_google_events(service):
+def list_all_google_events(service, settings: Settings | None = None):
+    settings = settings or load_settings()
 
     now = datetime.now().isoformat() + "Z"
     print(f"Getting the upcoming 100 events")
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)